ssg/constants.py

```diff
@@ -335,15 +340,15 @@
 }
 
 MULTI_PLATFORM_LIST = ["rhel", "fedora", "rhosp", "rhv", "debian", "ubuntu",
-                       "wrlinux", "opensuse", "sle", "ol", "ocp", "example"]
+                       "wrlinux", "opensuse", "sle", "ol", "ocp", "rhcos", "example"]
 
 MULTI_PLATFORM_MAPPING = {
     "multi_platform_debian": ["debian8", "debian9", "debian10"],
     "multi_platform_example": ["example"],
     "multi_platform_fedora": ["fedora"],
     "multi_platform_opensuse": ["opensuse"],
     "multi_platform_ol": ["ol7", "ol8"],
-    "multi_platform_ocp": ["ocp3", "ocp4"],
+    "multi_platform_ocp": ["ocp3", "ocp4", "rhcos4"],
```

Comment:
As you add `rhcos` to `MULTI_PLATFORM_LIST` in line 343 above, you probably mean to add `rhcos4` to a new entry: `multi_platform_rhcos`.

Reply:
uhm... well. rhcos is still just part of OCP. That's why I added it here :/

Reply:
> rhcos is still just part of OCP.

I understand. But creating a new product means a generating new DS.
And these are used to determine in which DS a piece of content should be included.

With this change, content marked with `multiple_platform_ocp` will also be included in `rhcos` DS, and given the current product naming this behavior may be confusing.

If there will be content that should only be included in `rhcos`, and will apply to multiple major versions of `rhcos`, `multi_platform_rhcos` makes sense.

Reply:
Got it. added it. Thanks!